config/locales/de.yml

```diff
@@ -1830,3 +1840,7 @@ de:
     zipcode: Postleitzahl
     zone: Gebiet
     zones: Gebiete
+  time:
+    formats:
+      solidus:
```

Comment:
This here is fine, though, since it's related to solidus.

Reply:
ok. this is needed for the order summary